whisper-script/reference-transcripts.py

```diff
@@ -0,0 +1,42 @@
+import whisper
```

Comment:
Consider changing filename to something like `extra-reference-transcripts.py` or `whisper-transcription.py` or `extra-whisper-transcription.py` so the name differentiates itself further from the base/required tooling.

Reply:
agreed - working on it